### PR TITLE
LLM: support Mistral AWQ models

### PR DESCRIPTION
**Pull request by plusbang**
## Description

Support Mistral AWQ models

### 3. Summary of the change 

- Add mistral support in src code
- Update CPU and GPU examples

### 4. How to test?
- [x] Unit test
- [x] Local test

### CODE REVIEW

File: python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/AWQ/mistral/generate.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#
```
Comment (jason-dai) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/AWQ/mistral/generate.py:1]: what's the difference vs. the LLaMA2 example?

Comment (plusbang) [reply python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/AWQ/mistral/generate.py:1]: > what's the difference vs. the LLaMA2 example?

The difference lies on prompt format and tokenizer class. Should we provide dedicated folder for each model, or just use one python script and list verified AWQ models in readme?

Comment (jason-dai) [reply python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/AWQ/mistral/generate.py:1]: Maybe one python script for CPU, one for GPU

Comment (plusbang) [reply python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/AWQ/mistral/generate.py]: > Maybe one python script for CPU, one for GPU

Have updated. As https://github.com/analytics-zoo/nano/issues/780#issuecomment-1824427117, the correctness is currently checked through observing the output text.

**Review by jason-dai — APPROVED** [review]
LGTM

**Pull request merged**